src/branding-bar/BrandingBar.js

```diff
@@ -1,40 +1,86 @@
+import useBreakpoint, { mediaQuery } from "@w11r/use-breakpoint";
 import React from "react";
+import useCollapse from "react-collapsed";
 import styled, { css } from "styled-components";
 
+import MenuClosedIconSrc from "../assets/icons/menuClosed.svg";
+import MenuOpenIconSrc from "../assets/icons/menuOpen.svg";
 import LogoIconSrc from "../assets/icons/recidiviz_logo.svg";
+import { FIXED_HEADER_HEIGHT, X_PADDING } from "../constants";
+import NavBar from "../nav-bar";
 import { LinkPill } from "../pill";
 
 const brandingBarFlexProperties = css`
   align-items: baseline;
   display: flex;
 `;
 
-const BrandingBarContainer = styled.header`
+const ICON_SIZE = "26px";
+
+const Y_MARGIN = "12px";
+
+const brandingBarFixedStyles = `
+  left: 0;
+  min-height: ${FIXED_HEADER_HEIGHT}px;
+  padding: 0 ${X_PADDING}px;
+  position: fixed;
+  right: 0;
+  top: 0;
+`;
+
+const BrandingBarWrapper = styled.header`
+  align-items: flex-start;
+  background: ${(props) => props.theme.colors.background};
   display: flex;
   flex-wrap: wrap;
   justify-content: space-between;
-  margin-bottom: 80px;
+
+  ${mediaQuery(["mobile-", brandingBarFixedStyles])}
 `;
 
 const BrandingBarHeader = styled.div`
   ${brandingBarFlexProperties}
+
+  ${(props) => (props.collapsible ? "align-items: center;" : "")}
+  margin-top: ${(props) => (props.expanded ? `${FIXED_HEADER_HEIGHT}px` : 0)};
+  transition: all
+    ${(props) =>
+      `${props.theme.transition.defaultDuration} ${props.theme.transition.defaultEasing}`};
+`;
+
+const Icon = styled.img`
+  height: ${ICON_SIZE};
+  width: ${ICON_SIZE};
 `;
 
-const Logo = styled.img``;
+const Logo = styled(Icon)`
+  margin: ${Y_MARGIN} 0;
+`;
 
 const BrandingBarTitle = styled.h1`
   color: ${(props) => props.theme.colors.heading};
   font: ${(props) => props.theme.fonts.display};
-  display: inline;
+  font-size: ${(props) =>
+    props.small
+      ? props.theme.fonts.brandSizeSmall
+      : props.theme.fonts.brandSizeLarge};
+  display: inline-block;
   margin-left: 16px;
+  opacity: ${(props) => (props.hide ? 0 : 1)};
+  transition: all
+    ${(props) =>
+      `${props.theme.transition.defaultDuration} ${props.theme.transition.defaultEasing}`};
+  width: ${(props) => (props.small ? "110px" : "auto")};
```

Comment:
Does this `width` need to be this small on small screens?  If we make it 200px for example, we can include the `SITE_TITLE` on smaller screens like so:

<img width="304" alt="Screen Shot 2020-07-16 at 4 55 15 PM" src="https://user-images.githubusercontent.com/25503/87721918-39a65000-c785-11ea-81a4-fb4e62a74b90.png">

I know that ultimately the `SITE_TITLE` will be dynamic which will mean we need to think about how to handle really long text, but for the North Dakota launch we should be ok.

Reply:
it doesn't have to be for technical reasons, I was just matching the mocks ... it is likely going to have to change at some point anyway once the title is finalized